Framework/Kernel/test/AttenuationProfileTest.h

```diff
@@ -25,6 +25,9 @@ class AttenuationProfileTest : public CxxTest::TestSuite {
 
   void testLoadAttenuationFile() {
     std::string path = Mantid::Kernel::ConfigService::Instance().getFullPath("AttenuationProfile.DAT", false, 0);
+    TS_ASSERT(!path.empty());
+    std::cout << "***" << path << "***\n";
+    auto profile = AttenuationProfile(path, "");
```

Comment:
Nice! Just want to check if we want to keep these two lines, or if they were for debugging?

Reply:
I removed that line. It was to decide if I had really not changed anything.